collector.py

```diff
@@ -0,0 +1,110 @@
+"""
+Collector script is a simple CLI tool to collect and persist data in Solr indexes.
+On the flag basis, user can select whether just show data or show + persis the data
+Usage ::
+    collector.py [OPTIONS]
+    collector.py --help | -h
+    collector.py --option
+        Explain - {option}
+            collector.py --option general_info phone whatsapp facebook
+    collector.py
+"""
+
+import sys
+
+from scripts import dbm
+from scripts.extract_all import extract_all_data
+from scripts.fb_reader import store_fb_data
+from scripts.report import REPORT
+from scripts.phone import store_phone_data
+from scripts.wa_reader import store_wa_data
+
+help_str = """
+Collector is a simple CLI tool to collect data from rooted android device and persist data for analysis 
+    collector.py [OPTIONS]
+    collector.py --help | -h
+    collector.py --option | -o <space separated options> --session_name <name_for_session_without_spaces>
+        Explain - {option}
+        0. all : collect all common dbs
+        1. general_info : To collect general device information
+        2. phone : To collect mobile contacts, call logs and messages information
+        3. whatsapp : To collect general whatsapp message information and contact details
+        4. facebook : To collect facebook messages and profile information
+            
+            collector.py --option general_info phone whatsapp facebook
+        
+        Explain - {session_name} - {No_Space-In-Name}
```

Comment:
Can you describe the use cases where session name will be required?

Reply:
@Wolf-Legend

Reply:
Sure, will add it there. Also, will be using click in improved version so these things will be selt documented :)